maro/simulator/scenarios/oncall_routing/business_engine.py

```diff
@@ -137,6 +137,7 @@ def __init__(
                 self._refresh_plan_duration(tick=-1, route_idx=route.idx, index=i)
 
         # Step 7: Create the carrier arrival events.
+        self._current_dest_arrival_tick: Dict[str, Optional[int]] = {}
```

Comment:
**If we want to keep this variable**, (although the result would be right) maybe it is better to clear this dict in reset()?